tests/unit_tests/conftest.py

```diff
@@ -101,7 +101,8 @@ def database(db_uri):
     """
     url = make_url(db_uri)
     db_to_create = url.database
-    url.database = "postgres"
+    # url.database = "postgres"
+    url = url.set(database="postgres")
```

Comment:
I'm not sure about this change.

Probably OK; the idea here is to temp. have access to postgres DB (which contains the user info of postgres); so the fixture can handle creating and destroying the test DB.

Reply:
SQL Alchemy 1.4 has the url immutable, that's why i am using url.set()